hoomd_benchmarks/write_gsd.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2021-2023 The Regents of the University of Michigan
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+"""Benchmark GSD writes."""
+
+import hoomd
+import time
+from . import common
+from .configuration.hard_sphere import make_hard_sphere_configuration
+
+DEFAULT_BANDWIDTH = 'false'
```

Comment:
Why is this being stored as a string and not a boolean?

Reply:
This is a typo. Fixed.